tests/unit/php/Core/Framework/Resources/config/test_entity_definition.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" ?>
```

Comment:
this file is never loaded and can probably be removed

Reply:
yes, they are dependency, which are not resolved in the compiler class.